Sources/StreamChatSwiftUI/ChatChannel/ChatChannelViewModel.swift

```diff
@@ -177,6 +177,20 @@ open class ChatChannelViewModel: ObservableObject, MessagesDataSource {
             object: nil
         )
         
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(onViewAppear),
+            name: NSNotification.Name(.sheetHiddenNotification),
+            object: nil
+        )
```

Comment:
What was the issue this code is fixing? Currently it is just sent by some sheets from poll attachment view but there are other sheets as well which do not send any notifications.

Reply:
fixes the sheet disappearing when adding reactions. Not sure if it's possible to fix it in another way, given the way SwiftUI works. The reactions would change the id of the message, thus changing its identity. With the fix, when there's a sheet open, we delay the update of the message.

Reply:
I see the problem now. Maybe we should rename the notification to be message specific. `.messageCellSheetHiddenNotification` or smt which tells that this is related to the message list only.

Reply:
I will document this.